### DISCUSSION TITLE
Quadrantsec integration

### DISCUSSION THREAD

**Discussion by hugollanos** [zinclabs/zinc#36] [General]
Hello! I read that Quadrantsec is using ZINC.... Is possible to use Sagan to get logs from ZINC? What is the integration?
Really interested in ZINC+Cibersecurity use case ;)

Thanks!

**Comment by prabhatsharma** [comment]
From [reddit](https://www.reddit.com/r/selfhosted/comments/r7y585/zinc_a_lightweight_alternative_to_elasticsearch/hnlhwsj/?context=3) the team is pushing sagan and suricata logs to Zinc.
